src/cmd/troubleshoot/image_helper.go

```diff
@@ -0,0 +1,125 @@
+package troubleshoot
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+var (
+	removeSchemaRegex      = regexp.MustCompile("^.*//(.*)$")
```

Comment:
nitpick: with regexes I think a comment is necessary to describe what they are doing. maybe with an example, so they don't have to be manually parsed when read

Reply:
I tried and failed. Figuring out exactly what those regexes do is urgently necessary, but out of scope here. This is part of a piece of code I just moved. The regex stuff in this file generally needs some more love.